src/schoology-extractor/tests/data_generation/__main__.py

```diff
@@ -17,6 +17,7 @@
 )
 from tests.data_generation.assignments import (
     generate_and_load_assignments,
+    generate_extra_assignments_without_enrollments,
```

Comment:
Its not being used, and thus causing a style check fail.

```suggestion
   # generate_extra_assignments_without_enrollments,
```